devel/resource_dasm/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+PortGroup               compiler_blacklist_versions 1.0
+
+github.setup            fuzziqersoftware resource_dasm 46ad32e
+github.tarball_from     archive
+
+compiler.blacklist-append {clang < 1300}
```

Comment:
This might need some tweaking depending on what C++20 features (if any) are used, but we'll see what happens on the buildbots.

Would you mind adding a comment above the blacklist in both ports mentioning that this is for C++20 support? Thanks.

Reply:
Thanks, I had meant to add such a comment, but neglected to add it before pushing.
Latest push addresses the dependency categories as well.